### ISSUE TITLE
[Word]: NVDA is not announcing any of the available content in the conflict changes pane.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8285]
###Steps to reproduce:
1. Open a shared doc 
2. Now make the changes in the same paragraph to get conflict.
3. Press Ctrl+S and observe that NVDA is not announcing the Upload failed message for the same. but there is some sound for that.
4. Navigate to resolve button and open the conflicting changes pane
5. Navigate in the pane using up/down arrow
6. Observe that NVDA is not announcing any of the available content in the pane.
 
### Expected behavior:
NVDA should announce the available content in the Conflict changes pane.

### Actual behavior:
NVDA is not announcing any of the content available in the conflicting changes pane.

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![conflict_changes](https://user-images.githubusercontent.com/22956130/40102076-2922d0d0-5907-11e8-9e20-4ef52fa98780.png)

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
@WiproPSAccTest are you still having this issue with NvDA 2024.2 or 2024.3 Beta?

**Comment by Adriani90** [comment]
cc: @msftedad can you please test whether this issue is still reproducible with NVDA 2024.4 RC?

**Comment by msftedad** [comment]
Hi @Adriani90, we are checking with the product team and will update you soon.

**Comment by msftedad** [comment]
Hi @Adriani90, product team confirmed it is not repro anymore. this issue will not repro because we do not have the conflicting changes/errors pane anymore. Now conflicting changes just show as track changes, and we auto merge them.

**Comment by Adriani90** [comment]
Thanks for the update, then I close as this became a non issue.
